requirements.txt

```diff
@@ -7,5 +7,4 @@ django-debug-toolbar==3.4.0
 djangorestframework==3.13.1
 djangorestframework-simplejwt==5.2.0
 drf-spectacular==0.22.1
-Pillow==9.1.1
-
+Pillow
```

Comment:
restore this change

Reply:
I changed the version because there was a conflict